src/components/organisms/AdminVenueView/components/VenueRoomsEditor/VenueRoomsEditor.tsx

```diff
@@ -265,14 +266,20 @@ export const VenueRoomsEditor: React.FC<VenueRoomsEditorProps> = ({
     ]
   );
 
+  const backgroundImageClass = `VenueRoomsEditor__background-image ${backgroundImageClassName}`;
```

Comment:
```suggestion
```

This is enough:
```tsx
          <img
            alt="draggable background"
            className="VenueRoomsEditor__background-image"
            src={backgroundImage}
          />
```

There is no need to pass separate class names here. If parent needs it to look different, in its own SCSS it can just use:

```scss
.ParentComponent {
  .VenueRoomsEditor__background-image {
    // whatever changes are needed
  }
}
```

Reply:
fixed in a4aa1c4f4ec90d75594111a0ed9e49045dfe7cd6